snscrape/modules/telegram.py

```diff
@@ -231,3 +311,23 @@ def _cli_setup_parser(cls, subparser):
 	@classmethod
 	def _cli_from_args(cls, args):
 		return cls._cli_construct(args, args.channel)
+
+def parse_num(s):
+	s = s.replace(' ', '')
+	if s.endswith('M'):
+		return int(float(s[:-1]) * 1e6), 10 ** (6 if '.' not in s else 6 - len(s[:-1].split('.')[1]))
+	elif s.endswith('K'):
+		return int(float(s[:-1]) * 1000), 10 ** (3 if '.' not in s else 3 - len(s[:-1].split('.')[1]))
+	else:
+		return int(s), 1
+
+def durationStrToSeconds(durationStr):
+	return sum([int(s) * int(g) for s, g in zip([1, 60, 360], reversed(durationStr))])
+
+def telegramResponseOkCallback(r):
+	if r.status_code == 200:
+		return (True, None)
+	elif r.status_code // 100 == 5:
+		return (False, f'status code: {r.status_code}')
```

Comment:
```suggestion
		return (False, f'{r.status_code=}')
```
discovered this recently for python 3.8+, see [here](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging)